src/app/ShoppingList.js

```diff
@@ -0,0 +1,100 @@
+import { MainMenu } from './MainMenu.js';
+import { MENU } from '../GlobalData.js'
+
+MainMenu(MENU.shoppingList);
+
+function getDishName(dishName) {
+    fetch(`https://api.spoonacular.com/recipes/autocomplete?number=3&query=${dishName}&apiKey=82b93f78e7ca49cfa0aed375729b37ce`)
+        .then(response => response.json())
+        .then(data => {
+
+            let output = ``;
+            data.forEach(element => {
+                output += ` <li><button data-id='${element.id}'>${element.title}</li>`; 
+            });
+            document.querySelector('.dish').innerHTML = output;
+            
+            document.querySelectorAll('.dish li button').forEach((e) => {
+                e.addEventListener('click', function() {
+                    console.log(e.textContent);
```

Comment:
Do we need this here? :-)

Reply:
console.log? One deleted, leaving one for the project requirements